src/common/options/global.yaml.in

```diff
@@ -6086,4 +6086,73 @@ options:
   services:
   - rgw
   - osd
-  with_legacy: true
\ No newline at end of file
+  with_legacy: true
+- name: bluestore_codel
+  type: bool
+  level: advanced
+  desc: enable/disable bluestore SlowFastCodel
+  default: false
+  with_legacy: true
+- name: bluestore_codel_throughput_latency_tradeoff
+  type: float
+  level: advanced
+  desc: adjust the tradeoff between throughput and bluestore latency in SlowFastCodel
+  long_desc: This parameter defines the amount of throughput loss (MB/s) for one ms
+    decrease in bluestore latency. (a value of 5 means that we are willing to lose
+    maximum of 5 MB/s of throughput for every 1 ms decrease in bluestore latency)
+  default: 5
+  with_legacy: true
+- name: bluestore_codel_initial_target_latency
+  type: int
+  level: advanced
+  desc: initial target latency for SlowFastCodel in ns
+  default: 5000000
+  with_legacy: true
+- name: bluestore_codel_slow_interval
+  type: int
+  level: advanced
+  desc: the interval of slow loop in SlowFastCodel in ns (this parameter should be larger that 'bluestore_codel_fast_interval')
+  default: 500000000
```

Comment:
How about converting time to seconds, or milliseconds but allow them to be floats?